tests/commands/test_new.py

```diff
@@ -0,0 +1,64 @@
+import subprocess
+import os
+import logging
+from pathlib import Path
+
+logging.basicConfig()
+logger = logging.getLogger('onyo')
+logger.setLevel(logging.INFO)
+
+test_dirs = ['simple',
+             's p a c e s',
+             's p a/c e s',
+             'r/e/c/u/r/s/i/v/e',
+             'relative',
+             'one',
+             'two',
+             'three',
+             'overlap/one',
+             'overlap/two',
+             'overlap/three',
+             'very/very/very/deep'
+             ]
+
+
+def populate_test_repo(path):
+    ret = subprocess.run(['onyo', 'init', path])
+    assert ret.returncode == 0
+
+    # enter repo
+    original_cwd = Path.cwd()
+    os.chdir(path)
+
+    # create dirs
+    ret = subprocess.run(['onyo', 'mkdir'] + test_dirs)
+    assert ret.returncode == 0
+
+    # return to home
+    os.chdir(original_cwd)
+
+
+def test_new_non_interactive():
+    populate_test_repo('testing_new')
+    os.chdir('testing_new')
```

Comment:
This is actually not necessary.

`change_cwd_to_sandbox(()` (in `conftest.py`) puts every test-file into its own "sandbox" directory. It's just a dir, so it still needs an `onyo init`, but you don't need to create yet another subdir inside of it if you're only going to use one.

You can simply `populate_test_repo('./')` and remove the `os.chdir` in all of the tests.